### PR TITLE
Add run_galaxy.sh

### PR DESCRIPTION
**Pull request by Marie59**
This file can come as a replacement of run.sh. The issue with run.sh is that some permissions are in comments because you say it's too slow. So I tried to do another version of the run.sh (run_galaxy.sh) that way maybe in the dockerfile you can add a variable ARG where we can decide which file to use either run.sh or run_galaxy.sh. And so, on the docker hub we will have 2 tags one with the run.sh version and one for the galaxy. What do you think of that ? 
For the run_galaxy.sh I took inspiration of this file https://github.com/anuprulez/ml-jupyter-notebook/blob/master/startup.sh.

### CODE REVIEW

**Comment by Marie59** [comment]
So for this one, for now in the Dockerfile it's using run.sh and for galaxy it needs run_galaxy.sh. Do we add a variable `ARG` before merging ?

**Comment by Marie59** [comment]
I added some permissions and I think on your run.sh you put them as comments because it was too slow. It's the main difference if I did everything right...

**Comment by ctroupin** [comment]
ok, so let's wait for the checks to finish, then I'll merge the PR.

**Comment by ctroupin** [comment]
True, I forgot about that.
I'll re-run the checks after then.

**Review by ctroupin — APPROVED** [review]

**Comment by Marie59** [comment]
Okay everything seems to be working !

**Comment by Marie59** [comment]
Do you have something already in place to push this docker image on dockerhub ? Or should I do something ?

**Comment by Alexander-Barth** [comment]
I just pushed a new image. Do you see the tag 2023-04-18T0659 ?

https://hub.docker.com/layers/abarth/divand-jupyterhub/2023-04-18T0659/images/sha256-a836068df5b558e4d80e2468a0a4ab89b0b94a808d92181d1719279bfa6c34ba?context=repo

**Pull request merged**
